INSTALL.md

```diff
@@ -199,3 +199,7 @@ https://aur.archlinux.org/packages/rofi-lbonn-wayland-git/
 ### Fedora
 
 https://copr.fedorainfracloud.org/coprs/alebastr/sway-extras/
+
+### NixOS
+
+Install	[nur.repos.c0deaddict.rofi-wayland]([https://github.com/nix-community/nur-combined/blob/master/repos/c0deaddict/pkgs/rofi-wayland/unwrapped.nix) from NUR.
```

Comment:
The link seems broken (extra `[`) and please also remove the extra spaces after "Install"